data/apollo/apollo.debian.yaml

```diff
@@ -0,0 +1,22 @@
+identifiers:
+  - biotools: apollo
+License: Artistic
```

Comment:
Small letter for consistence, and sorted out of `identifiers`

Reply:
Solved by @smoe in 5713268. Note: The present licenses are only for non-official packages.